codemeta.json

```diff
@@ -1,703 +1,372 @@
 {
     "@context": "https://doi.org/10.5063/schema/codemeta-2.0",
     "@type": "SoftwareSourceCode",
+    "codeRepository": "https://github.com/gammapy/gammapy",
+    "description": "Gammapy analyzes gamma-ray data and creates sky images, spectra and lightcurves, from event lists and instrument response information; it can also determine the position, morphology and spectra of gamma-ray sources. It is used to analyze data from H.E.S.S., Fermi-LAT, HAWC, and the Cherenkov Telescope Array (CTA).",
```

Comment:
not MAGIC?

Reply:
Do we have a science paper with MAGIC data?

Reply:
Ah probably not! @cosimoNigro ?

Reply:
I put the names of experiments for which Science paper have been published using Gammapy. And I have not found a paper wtith a an analysis with Gammapy of MAGIC or VERITAS data. They are papers that analyse Fermi-LAT data with Gammapy